app/src/main/java/org/sopt/cgv/feature/time/component/TheaterClassificationTabInModal.kt

```diff
@@ -0,0 +1,79 @@
+package org.sopt.cgv.feature.time.component
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Tab
+import androidx.compose.material3.TabRow
+import androidx.compose.material3.TabRowDefaults
+import androidx.compose.material3.TabRowDefaults.tabIndicatorOffset
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.persistentListOf
+import org.sopt.cgv.core.designsystem.theme.CGVTheme
+import org.sopt.cgv.core.designsystem.theme.Gray850
+import org.sopt.cgv.core.designsystem.theme.PrimaryRed400
+import org.sopt.cgv.core.designsystem.theme.White
+
+@Composable
+fun TheaterClassificationTabInModal(
+    selectedTabInModalIndex: MutableState<Int>,
```

Comment:
```suggestion
    selectedTabIndex: Int,
    onTabSelected: (Int) -> Unit
```
여기서도 MutableState를 직접 전달하는 대신 불변 값(Int)과 이벤트 콜백을 전달하면
상태 변경 로직이 컴포넌트 외부로 이동하여 **컴포넌트는 상태 표시와 이벤트 전달만 담당**할 수 있을 거 같아요!